SimCore/src/SimCore/XsecBiasingOperator.cxx

```diff
@@ -6,7 +6,7 @@ namespace simcore {
 
 XsecBiasingOperator::XsecBiasingOperator(
     std::string name, const framework::config::Parameters& parameters)
-    : G4VBiasingOperator(name) {}
+    : G4VBiasingOperator(name), theLog_{framework::logging::makeLogger(name)} {}
```

Comment:
I think you merged but should have rebased, bc these changes here are already merged in https://github.com/LDMX-Software/ldmx-sw/pull/1630/files

Reply:
```
git switch trunk
git fetch
git pull
git switch -
git rebase trunk
```